components/Util.js

```diff
@@ -117,7 +117,8 @@ export const sortData = (data, sortValue) => {
 
 const valueTypesIterator = (operator, value, isSubstringSearch = false, isEqualOperator = false ) => {
   const valueTypes = ["value_string", "value_long_string", "value_double"];
-  const isNotCovertible = isNumber(value);
+  const isBoolean = (value === "true" || value === "false")
```

Comment:
Suggestion: export this as a function, so that it can be reused in other contexts.
```suggestion
  export const isBoolean = (value) => (value === "true" || value === "false")
```